README.md

```diff
@@ -5,12 +5,15 @@
 [![badge-discord][]][link-discord] [![badge-matrix][]][link-matrix] [![badge-opencollective][]][link-opencollective]
 
 Stream VR games from your PC to your headset via Wi-Fi.  
-ALVR uses technologies like [Asynchronous Timewarp](https://developer.oculus.com/documentation/native/android/mobile-timewarp-overview) and [Fixed Foveated Rendering](https://developer.oculus.com/documentation/native/android/mobile-ffr) for a smoother experience.  
-Most of the games that run on SteamVR or Oculus Software (using Revive) should work with ALVR.  
 This is a fork of [ALVR](https://github.com/polygraphene/ALVR).
 
+### Direct download the lastest version:
+### [Windows Launcher](https://github.com/alvr-org/ALVR/releases/latest/download/alvr_launcher_windows.zip) | [Linux Launcher](https://github.com/alvr-org/ALVR/releases/latest/download/alvr_launcher_linux.tar.gz)
```

Comment:
I think this should have some sort of reference to still checking the wiki, because people won't scroll as far in the readme if they use this, and we don't need even more people not checking the wiki when requesting help

Reply:
I'm conflicted on this. There are also people that couldn't figure out where to download, go figure. We should decide the best order of sections in the readme.

Reply:
Tbh I don't know of a way to shove in a short reference to the wiki into that section and the section till the wiki in the readme isn't that long either, so let's just leave it as is